instance/jmsml_D_Sea_Subsurface.xml

```diff
@@ -379,33 +379,57 @@
     </LegacySymbol>
     <LegacySymbol ID="MILITARY_SUBMARINE_POSSIBLE_SUBMARINE_LOW_1_MOD_SYM" Label="S*UPS1----*****" EntityID="MILITARY" EntityTypeID="SUBMARINE" ModifierOneID="POSSIBLE_SUBMARINE_LOW_1_MOD">
       <LegacyFunctionCode Name="2525C">S1----</LegacyFunctionCode>
+	  <LegacyEntity ID="MILITARY_SUBMARINE_POSSIBLE_SUBMARINE_LOW_1_MOD_LE" Label="WAR.SBSUF.SUB.POSS1" Graphic ="S-U-S1---.svg">
```

Comment:
@BobBooth - spaces usually aren't used, for consistency, between the attribute name and its value.  For example Graphic ="S-U-S1----.svg" should be Graphic="S-U-S1----.svg".